### PR TITLE
chore(Theme): replace darkMode with `colorScheme` prop

### PR DESCRIPTION
**Pull request by tujoworker**
Add colorScheme prop with values auto, light, dark, and inherit. 

It was not documented, so no breaking change.

### CODE REVIEW

File: packages/dnb-eufemia/src/shared/__tests__/Theme.test.tsx [hunk 1]
```diff
@@ -86,16 +86,134 @@ describe('Theme', () => {
...
+      </Theme>
+    )
+
+    const parent = document.querySelector('#parent')
+    const child = document.querySelector('#child')
+    expect(Array.from(parent.classList)).toContain(
+      'eufemia-theme__color-scheme--dark'
+    )
+    expect(Array.from(child.classList)).toContain(
+      'eufemia-theme__color-scheme--inherit'
```
Comment (langz) [inline packages/dnb-eufemia/src/shared/__tests__/Theme.test.tsx]: If it inherits colorScheme from parent, shouldn't this be `--dark` then, instead of `--inherit`?

**Review by langz — APPROVED** [review]

**Comment by langz** [comment]
Probably just an edge-case, but which CSS class does one get if rendering a single `<Theme colorScheme="inherit" />` ?

**Comment by tujoworker** [comment]
I think we should remove it actually. From a CSS point of view, it makes no sense.
